### PR TITLE
[#2144] Install Minio 7.1.16 for OS using Python version earlier than 3.7 (main)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: irods_consortium_continuous_integration_test_hook.py [hunk 1]
```diff
@@ -34,7 +34,14 @@ def get_package_type():
 def install_test_prerequisites():
     irods_python_ci_utilities.subprocess_get_output(['sudo', 'python3', '-m', 'pip', 'install', '--upgrade', 'pip>=20.3.4'], check_rc=True)
     irods_python_ci_utilities.subprocess_get_output(['sudo', 'python3', '-m', 'pip', 'install', 'boto3', '--upgrade'], check_rc=True)
-    irods_python_ci_utilities.subprocess_get_output(['sudo', 'python3', '-m', 'pip', 'install', 'minio', '--upgrade'], check_rc=True)
+
+    # Minio 7.1.17 imports the annontations module which only exists in Python 3.7 and beyond.
+    # CentOS 7 defaults to Python 3.6, therefore we have to install the previous version of Minio
```
Comment (alanking) [inline irods_consortium_continuous_integration_test_hook.py:39]: Is this also the case for Ubuntu 18.04?

Comment (korydraughn) [reply irods_consortium_continuous_integration_test_hook.py:39]: If Ubuntu 18 ships with Python 3.6, then yes.

We'll hold on merging until we confirm.

Comment (korydraughn) [reply irods_consortium_continuous_integration_test_hook.py:39]: This also affects Almalinux 8.

Comment (korydraughn) [reply irods_consortium_continuous_integration_test_hook.py:39]: Definitely. (b) looks like the best approach. We know this issue is specific to python 3.6.

Comment (korydraughn) [reply irods_consortium_continuous_integration_test_hook.py]: Done.

**Comment by korydraughn** [comment]
Running test to verify correctness.

**Review by alanking — APPROVED** [review]
Squash and # once confirmed.

**Comment by korydraughn** [comment]
#'d

**Review by alanking — APPROVED** [review]

**Pull request merged**